src/trace/Kconfig

```diff
@@ -88,4 +88,11 @@ config TRACE_BURST_COUNT
 		Amount of messages that will pass through the filter even if sent in rapid succession.
 		Allowed message burst size before filter suppresses the message.
 
+config TRACE_DMA_DISABLE
```

Comment:
Should be TRACE_DMA_ENABLE defaulting to `y` to avoid double negations everywhere (even more confusing when combined with #elif)

Reply:
Sure, I'd prefer that too, but you cannot "select TRACE_DMA_ENABLE=n" so you'd have to add "select TRACE_DMA_ENABLE" to all platforms but one. And in general - does it make sense to have a not user-selectable Kconfig option, defaulting to y? You'd have no way to unselect it.

Reply:
That does not seem correct:
```shell
git grep '=n' -- src/arch/xtensa/configs/ | wc -l
77
```

Reply:
@marc-hb Sure, those are user-selectable Kconfig options. I don't think you can set non user-selectable options in defconfigs